todo.js

```diff
@@ -15,3 +15,39 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+const fs = require('fs');
+let todoList = fs.readFileSync("todos.txt","utf-8");
+let command = process.argv[2];
+let readLineSync = require("readLine-sync");
+let TODO = {};
+function list() {
+
+  if (command === "list")
+  {
+    console.log(todoList);
+  }
+}
+list();
+function append() {
+  if (command === "append")
+  {
+    let newTask = readLineSync.question("What would you like to add?");
```

Comment:
See the README for how this feature is supposed to work. Let me know if you have questions about command line arguments.